src/main/java/org/veupathdb/service/eda/ss/model/db/FilteredResultFactory.java

```diff
@@ -127,6 +133,38 @@ public static void produceTabularSubset(DataSource dataSource, String appDbSchem
     }, FETCH_SIZE_FOR_TABULAR_QUERIES);
   }
 
+  /**
```

Comment:
would be nice if comment indicates that this is the top level of a recursive map-reduce blah blah blah